main.css

```diff
@@ -1,370 +0,0 @@
-#body{
-  position: static;
-}
-#header {
-  position: static;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
-a {
-  color: inherit;
-  text-decoration: none;
-}
-/*  - - - - fonts - - - - */ 
-.text-large {
-  font-size: 40px;
-  font-family: Roboto, Courier;
-}
-.text-medium {
-  font-size: 20px;
-  font-family: Roboto, Courier;
-  font-weight: 300;
-}
-.text-small {
-  font-size: 16px;
-  font-family: Roboto, Courier;
-}
-/*  - - - - fonts end - - - - */ 
-/*  - - - - buttons - - - - */ 
-#top-button {
-  font-size: 16px;
-  margin: 30px;
-}
-#content-button {
-  font-size: 16px;
-  margin: 15x;
-}
-#footer-button {
-  font-size: 16px;
-  margin-top: 10px;
-  margin-right: 70px;
-  justify-content: left;
-}
-.buttonL {
-  background-color: rgb(95, 97, 95); 
-  border: none;
-  font-family: Roboto, Courier;
-  color: white;
-  padding: 10px 25px;
-  text-align: center;
-  display: inline-block;
-  cursor: pointer;
-  text-decoration: none;
-}
-.buttonS {
-  background-color: rgb(0, 255, 255); 
-  border: none;
-  font-family: Roboto, Courier;
-  color: white;
-  display: inline-block;
-  text-align: center;
-  width: 35%;
-  cursor: pointer;
-  border-radius: 5px;
-  text-decoration: none;
-}
-.buttonL:hover {
-  cursor: pointer;
-  opacity: 65%;
-}
-.buttonS:hover {
-  cursor: pointer;
-  opacity: 65%;
-}
-/*  - - - - buttons end - - - - */ 
-
-/*  - - - - top section, main page - - - - */ 
-#linediv {
-  display: flex;
-  justify-content: center;
-  margin-top: 100px;
-  margin-bottom: 100px;
-}
-#top-line {
-  width: 600px;
-}
-.line {
-  border-top: 1px solid grey;
-  border-bottom: 1px solid grey;
-  height: 2px;
-}
-#features{
-  position: static;
-}
-#title-desc {
-  padding-top: 10px;
-}
-#top-photos {
-  display: flex;
-  align-items: center;
-  flex-direction: column; 
-  margin: auto;
-}
-#photos-div {
- display: grid;
- grid-template-columns: 1fr 1fr 1fr;
- grid-template-rows: 90% 10%;
- text-align: center;
- grid-template-areas: 
- "top-cage top-cage top-cage"
- "text-small text-small text-small";
-}
-.top-cage {
-  display: flex;
-  align-items: center;
-  flex-direction: row; 
-  justify-content: space-between;
-  margin: 20px
-}
-
-/*  - - - - quote box - - - - */ 
-#quote-box {
-  background-image: url("Images/parralax.jpg");
-  height: 200px;
-  background-attachment: fixed;
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: cover;
-  margin-bottom: 50px;
-}
-#center-quote {
-  vertical-align: middle;
-  text-align: center;
-  line-height: 200px;
-  font-style: italic;
-}
-#quote-text {
-    position: absolute;
-    z-index: 999;
-}
-/* - - - quote box end - - - - */ 
-
-/* - - - main content - - - - */ 
-#main {
-  width: 75%;
-  display: grid;
-  grid-template-columns: 2fr 1fr;
-  grid-template-rows: 1fr;
-  grid-template-areas: 
-    "left-content right-content";
```

Comment:
nice use of grid, template-areas, and classes to wrap all of your left-content and all of your right-content.

Reply:
It does seem like you're going for a one dimensional layout here, so FlexBox might by a simpler tool to use.